packages/manager/modules/pci/src/components/order-command/order-command.scss

```diff
@@ -0,0 +1,5 @@
+.overloader_height_tail_logs {
+  .tail-logs {
+    height: auto !important;
```

Comment:
**nitpick:** FYI we tend to avoid relying on custom style.

If you noticed an issue on `tail Logs` component, please report it via an Issue or you can submit a Pull Request.

Reply:
I do agree with you and I try to avoid this as much as possible. Here however, I feel the issue is not with tail-logs per se but with the fact that I'm not sure that this component should have been used in the first place. 
The way it is now, it displays a double vertical scroll because tail-logs has it's height fixed (which makes sense as it's supposed to display rolling logs).
To avoid this, I basically had 3 options :